src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/Example1.java

```diff
@@ -0,0 +1,60 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="UnnecessaryParentheses"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessaryparentheses;
+
+import java.util.List;
+
+// xdoc section -- start
+class Example1 {
```

Comment:
```suggestion
class Example1 {

```

Reply:
@nrmancuso these changes make the example file length more than 60 lines.so I didn't apply them cause this is a violation `Example1.java:1: File length is 64 lines (max allowed is 60). [FileLength]` So please If you have another solution for this make a review for it

Reply:
@mahfouz72 thanks for bringing this up. I personally find this example to be hard to follow with no empty lines to separate the methods. @rnveach thoughts on extending file length limit for examples?